src/frontend/next/src/pages/myfeeds.tsx

```diff
@@ -0,0 +1,200 @@
+import { useState, useEffect, FormEvent } from 'react';
+import { useRouter } from 'next/router';
+import { Box, Card, Container, Grid, IconButton, Typography } from '@material-ui/core';
+import { AccountCircle, AddCircle, RssFeed } from '@material-ui/icons';
+import { makeStyles } from '@material-ui/core/styles';
+import { ValidatorForm, TextValidator } from 'react-material-ui-form-validator';
+import { isWebUri } from 'valid-url';
+import Head from 'next/head';
+
+import config from '../config';
+import ExistingFeedList from '../components/ExistingFeedList';
+import HelpPopoverButton from '../components/HelpPopoverButton';
+import CustomizedSnackBar from '../components/SnackBar';
+import { useUser } from '../components/UserProvider';
+import { User, Feed, FeedHash } from '../interfaces';
+
+const useStyles = makeStyles((theme) => ({
+  margin: {
+    margin: theme.spacing(2),
+  },
+  button: {
+    padding: '3px 0 3px 0',
+  },
+}));
+
+const MyFeeds = () => {
+  const router = useRouter();
+  const user: User = useUser();
+  const classes = useStyles();
+  const [newFeedAuthor, setNewFeedAuthor] = useState('');
+  const [newFeedUrl, setNewFeedUrl] = useState('');
+  const [submitStatus, setSubmitStatus] = useState({ message: '', isError: false });
+  const [feedHash, updateFeedHash] = useState({});
```

Comment:
A feed hash is a string, is this named incorrectly?

Reply:
@humphd : I believe the naming is incorrect. It is supposed to be like an object of keys which are feedHashes. For example:
```
{
    'absbcghdgsbds' : { some data},
    'sdadas12312ds' : {some data},
    .... 
}
```
I am thinking to change it to `feedHashObject` or sth